aws/ci_build

```diff
@@ -104,6 +109,7 @@ def main()
 
       HipChat.log message, color: 'red'
       HipChat.log "/quote #{log}", color: 'gray', message_format: 'text'
+      HipChat.log "/quote #{status}", color: 'gray', message_format: 'text'
```

Comment:
Why are we adding new logging to HipChat? Should there be a TODO to convert this logging to Slack?